.github/workflows/meta-balena-esr.yml

```
@@ -94,7 +94,7 @@ jobs:
           git fetch --tags origin
           current_os_version=$(git describe --abbrev=0 "$(git rev-list --tags --max-count=1)")
           current_os_version="${current_os_version:1}"
-          va=("${current_os_version//./ }")
+          va=(${current_os_version//./ })
```

```suggestion
          IFS="." read -r -a va <<< "${current_os_version}"
```
https://www.shellcheck.net/wiki/SC2206